src/Test/Utilities/Portable/CommonTestBase.CompilationVerifier.cs

```diff
@@ -88,7 +88,7 @@ internal ImmutableArray<ModuleMetadata> GetAllModuleMetadata()
                 return modules;
             }
 
-            public void Emit(string expectedOutput, IEnumerable<ResourceDescription> manifestResources, EmitOptions emitOptions, bool peVerify, SignatureDescription[] expectedSignatures)
+            public void Emit(string expectedOutput, int? expectedReturnCode, string[] args, IEnumerable<ResourceDescription> manifestResources, EmitOptions emitOptions, bool peVerify, SignatureDescription[] expectedSignatures)
```

Comment:
>Emit [](start = 24, length = 4)

It looks like we are not executing the program and not checking expectedReturnCode when expectedReturnCode is not null, but expectedOutput is. It feels like, either we should ensure that expectedOutput is not null in that case, or should compare the expectedReturnCode regardless whether expectedOutput is null or not.